elyra/pipeline/processor.py

```diff
@@ -187,6 +188,57 @@ def log_pipeline_info(self, pipeline_name: str, action_clause: str, **kwargs):
 
             self.log.info(f"{self._type} '{pipeline_name}'{op_clause} - {action_clause} {duration_clause}")
 
+    @staticmethod
+    def _propagate_operation_inputs_outputs(pipeline: Pipeline, sorted_operations) -> None:
```

Comment:
This could use a type hint.  Perhaps `sorted_operations: List[Operation]`?  You'd need `from typing import List`.

Or you could probably just go with `sort_operations: list`.

Reply:
Actually, whatever is used should match the return value of `_sort_operations()`. :smile: